### PR TITLE
change filter country design

### PR DESCRIPTION
**Pull request by spektra2147**

### CODE REVIEW

**Review by diashalabi — APPROVED** [review]

**Comment by fatihalp** [comment]
make it bigger

**Review by fatihalp — APPROVED** [review]

File: addons/default/visiosoft/advs-module/resources/css/list.css [hunk 1]
```diff
@@ -212,4 +212,8 @@ a.sort-by-open-dropdown:hover {
 .sort-by-item:hover {
     text-decoration: underline;
     cursor: pointer;
+}
+
+.filter-section .countries input[type=checkbox] {
+    display: none;
 }
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/css/list.css:219]: don't hide it, remove it, you can select one country why use checkbox

**Review by diashalabi — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/location-module/resources/views/ads-list/partials/location-fields.twig [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
+                data-toggle="modal">
+            <span class="float-left">
+                {{ trans("visiosoft.module.location::field.country.name") }}
+                    <small class="text-muted selected-country">{{ countryName }}</small>
+            </span>
+            <i class="fas fa-sort-down float-right"></i>
+        </button>
         <input name="country" value="{{ countryValue }}" type="hidden">
     </div>
+
```
Comment (fatihalp) [inline addons/default/visiosoft/location-module/resources/views/ads-list/partials/location-fields.twig:26]: unnecassary spaces

File: addons/default/visiosoft/advs-module/resources/css/list.css (hunk 1 above)
Comment (fatihalp) [reply addons/default/visiosoft/advs-module/resources/css/list.css:219]: it's not logical right but next week will be multiple selectcable like alibaba it cloud stay like that @diashalabi

**Review by diashalabi — APPROVED** [review]

**Pull request merged**